examples/jenkins-master/jenkins-slave-template.json

```diff
@@ -0,0 +1,114 @@
+
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "s2i-to-jenkins-slave",
+    "annotations": {
+      "description": "Provide convenient way to convert any S2I builder to a Jenkins slave image",
```

Comment:
it doesn't have to be an s2i builder, right?  it just has to be a base image that has the necessary components for doing whatever type of build you're trying to do.  the assemble/run scripts aren't even used, right?